doc/install.text

```diff
@@ -0,0 +1,124 @@
+The following notes should allow you to install the LSST stack and the PFS DRP.
+After the installation procedure example commands are given to show you how to
+use the pipeline. The commands have been tested on an Arch Linux machine as well
+as on Mac OS X.
+
+* Install lsst binary distribution (https://pipelines.lsst.io/install/conda.html)
```

Comment:
Honestly I don't know, but I believe that it shouldn't make a difference. Where did you setup that ubuntu machine where you tested the gcc (new version) bug? I could try to install miniconda there and try it out...